cmd/verifier/README.md

```diff
@@ -0,0 +1,30 @@
+# Verifying provenances
+
+The [`verifier`](/internal/verifier/) package provides functionality for verifying an input
+provenance file. The provenance file should follow the
+[Amber provenance](./../pkg/amber/schema/v1/provenance.json) format and provide a list of materials
+(including the source code and the build toolchain), and steps for building a binary from the listed
+materials. The verification logic uses the provenance file to build a binary, and checks that the
+binary has a SHA256 hash equal to the expected digest given in the provenance file.
+
+To verify a SLSA provenance of the Amber build type run:
+
+```bash
```

Comment:
The snippet below is not valid bash, because of the `$`. Either remove the `$`, or change the type to `console` (in which case you can also interleave $-prefixed commands with the corresponding output, if you like).

Reply:
Nice. I did not know about the `console` style.